catalog/core/catalog-core-impl/filter-proxy/src/test/java/ddf/catalog/filter/proxy/builder/test/FilterBuilderTest.java

```diff
@@ -770,7 +770,7 @@ public Object visit(Within filter, Object data) {
     public void test() {
 
         FilterBuilder builder = new GeotoolsFilterBuilder();
-        Filter filter = builder.allOf(
+        builder.allOf(
```

Comment:
this test looks incomplete... maybe we should evaluate filter instead of throwing it away